README.rst

```diff
@@ -32,6 +32,26 @@ settings.
    add ``"google_maps"`` to the policy value list.
 3. Click the "Save changes" button.
 
+Adding Google Maps API key
+--------------------------
+
+This Xblock requires a Google Maps API key to work properly. Visit
+https://developers.google.com/maps/documentation/javascript/get-api-key
+for more instructions about Google Maps API keys.
+
+After acquiring an API key enable the following APIs in Google Developers Console:
```

Comment:
`After acquiring the API key, enable the following APIs in the Google Developers Console:`

Also, provide a link to the console.